docs/content/guides/getting-started/setting-options.md

```diff
@@ -708,136 +811,25 @@ hot.setCellMeta(1, 1, 'readOnly', false);
 // returns `false`
 hot.getCellMeta(1, 1).readOnly;
 ```
-
-## Implementing custom logic
```

Comment:
I don't think this section should be removed.

I've seen that in https://github.com/handsontable/handsontable/pull/9787#issuecomment-1222006939 you're saying that it's a word-by-word duplicate of the other sections, but I don't think it's true.

The `cells` config option is mentioned only in the `Setting row options` section and used to define settings for rows, but apart from the callback argument list, I don't think there are any hints of it being able to target individual cells.

This entire section explains in (too much) detail how to do it, plus it mentions some other important information and performance tips.

There are, for sure some duplicates in the content, but I think it should be tackled outside of this task.

Reply:
will reremove